tailwind.config.ts

```diff
@@ -14,6 +14,7 @@ const config: Config = {
         'gradient-conic':
           'conic-gradient(from 180deg at 50% 50%, var(--tw-gradient-stops))',
         linearGradient: 'linear-gradient(90deg, #1ED5A9 0%, #01B4E4 100%)',
+        buttonTab: 'linear-gradient(90deg, #C0FECF 0%, #1ED5A9 100%)',
```

Comment:
we can do gradients using classes easily. do you see this buttonTab gradient reusable?
If not then we should fallback to using gradients with classes like bg-gradient-to-r  from-lightGreen to-lightBlue